lumicks/pylake/kymotracker/detail/peakfinding.py

```diff
@@ -184,13 +185,15 @@ def refine_peak_based_on_moment(
     time_points : array_like
         Time points at which the coordinate estimates were made.
     half_kernel_size : int
-        Half of the kernel size in pixels. The kernel is used to refine the line estimate. The kernel size used for this
-        refinement will be 2 * half_kernel_size + 1.
+        Half of the kernel size in pixels. The kernel is used to refine the line estimate. The
+        kernel size used for this refinement will be 2 * half_kernel_size + 1.
     max_iter : int
         Maximum number of iterations
     eps : float
-        We add a little offset to the normalization to prevent divisions by zeros on pixels that did not have any photon
-        counts. Eps sets this offset.
+        We add a little offset to the normalization to prevent divisions by zeros on pixels that
+        did not have any photon counts. Eps sets this offset.
+    bias_correction : bool
+        Apply a bias correction at the end of the centroid refinement?
```

Comment:
uncommon to read a question in a docstring.

better use something like `whether to apply...`

Reply:
Good point. Fixed.